02-oop-inheritance/js/indexOOPClass.js

```diff
@@ -0,0 +1,256 @@
+
+// Practice - Point 7.
+var MoviesModule = (function() {
```

Comment:
This wasn't necessary, but interesting thinking. In another way, you could have on the global block, the Movie and DownloadableMovie, and just setting them into MoviesModule, being an object. The benefit of the IIFE, aside that you don't need to invoke it, is that it gives you some privacy, but in this case I'm not sure we need it.